Casks/ccleaner.rb

```diff
@@ -1,6 +1,6 @@
 cask "ccleaner" do
-  version "1.18.30"
-  sha256 "52295172b846c477f1c5396c9f568d602b89fad52d2fcaba394a14296e096ce6"
+  version "2.4.141"
+  sha256 "3369595d8e8fce701a22a18df88632556ec81ffae98872c37e100754afe1fdcb"
 
   url "https://download.ccleaner.com/mac/CCMacSetup#{version.major_minor.no_dots}.dmg"
```

Comment:
```suggestion
  url "https://download.ccleaner.com/mac/CCMacSetup#{version.major}#{version.minor.rjust(2, "0")}.dmg"
```
The minor part of the `version` seems to be represented in the `url` as a two-digit number, which requires left-padding with zero here. Done like this in `bartender` as well.

Reply:
Oh nice thanks for the suggestion. 👍

Reply:
Hope it works, didn't try locally.